examples/llm.gnd

```diff
@@ -0,0 +1,11 @@
+let args
+debug _ "Our input is:" args
+return _ This is test
```

Comment:
This line was left from return primitive (#6) manual tests, and should not exist.